Assets/MixedRealityToolkit-SDK/Features/Input/InputSystemGlobalListener.cs

```diff
@@ -13,18 +12,15 @@ namespace Microsoft.MixedReality.Toolkit.SDK.Input
     /// </summary>
     public class InputSystemGlobalListener : MonoBehaviour
     {
-        private static IMixedRealityInputSystem inputSystem = null;
-        protected static IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
-
         private bool lateInitialize = true;
 
-        protected readonly WaitUntil WaitUntilInputSystemValid = new WaitUntil(() => InputSystem != null);
+        protected readonly WaitUntil WaitUntilInputSystemValid = new WaitUntil(() => MixedRealityManager.InputSystem != null);
 
         protected virtual void OnEnable()
         {
-            if (MixedRealityManager.IsInitialized && InputSystem != null && !lateInitialize)
+            if (MixedRealityManager.IsInitialized && MixedRealityManager.InputSystem != null && !lateInitialize)
             {
-                InputSystem.Register(gameObject);
+                MixedRealityManager.InputSystem.Register(gameObject);
```

Comment:
Not checking for no Input system
```
MixedRealityManager.InputSystem?.
```
Update and check throughout

Reply:
We check on the line above.